ansible/roles/software/common/php-common/files/phpreload

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+service php{{ php_version }}-fpm reload
```

Comment:
This won't get templated, since we're using the `copy` module. You'll want to move this to the `templates` dir and update the Ansible task.

Reply:
Good catch! Pushed a fix.